src/main/groovy/org/mockito/release/internal/gradle/BaseJavaLibraryPlugin.java

```diff
@@ -33,13 +35,15 @@
  * <ul>
  *     <li>Automatically includes "LICENSE" file in all jars.</li>
  *     <li>Adds build.dependsOn "publishToMavenLocal" to flesh out publication issues during the build</li>
+ *     <li>Makes pom-generating task depend on "fetchAllProjectContributorsFromGitHub" to load all contributors from GitHub</li>
  * </ul>
  */
 public class BaseJavaLibraryPlugin implements Plugin<Project> {
 
     private final static Logger LOG = Logging.getLogger(BaseJavaLibraryPlugin.class);
 
-    final static String PUBLICATION_NAME = "javaLibrary";
+    final static String PUBLICATION_NAME = "JavaLibrary";
```

Comment:
We already capitalize the name so we can keep publication name as "javaLibrary". It is consistent with Gardle domain object convention names.

"JavaLibrary" -> "javaLibrary"

I'll make a quick fix and push it to your branch :)